src/test/regress/expected/failure_create_reference_table.out

```diff
@@ -39,10 +39,10 @@ SELECT citus.mitmproxy('conn.onCommandComplete(command="BEGIN").kill()');
 (1 row)
 
 SELECT create_reference_table('ref_table');
-WARNING:  connection not open
-CONTEXT:  while executing command on localhost:9060
 ERROR:  connection error: localhost:9060
-DETAIL:  connection not open
```

Comment:
Do we see the same change if unified executor is disabled? If not, will we need to create 2 output files when merging to master? If yes, then what do you think about creating two output files now?